### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by czubair**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by JeffreyRichter — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
...
-- [ ] Removing API version(s) in a stable version
-- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
-- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
-
-**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+If you have any breaking changes as defined in the [Breaking Change Policy](http://aka.ms/bcforapi), request approval from the Breaking Change Review Board.
+  
+**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
```
Comment (weshaggard) [inline .github/PULL_REQUEST_TEMPLATE.md:61]: Do we really want to link to internal only documentation here?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+
+
+## ARM API Information (Control Plane)
+<details>
```
Comment (weshaggard) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: Do we really want to these to be collapsed sections? I worry a little that folks will not ever expand them and read or check any of the options.

Comment (czubair) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: Yeah, I see that point. @markweitzel any thoughts?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (czubair) [reply .github/PULL_REQUEST_TEMPLATE.md:61]: @JeffreyRichter are you okay with using these links or do you find it problematic?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (czubair) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: The collapsible sections do have the advantage of focusing people's attention on ARM vs Data plane until we are able to have 2 templates.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (JeffreyRichter) [reply .github/PULL_REQUEST_TEMPLATE.md:61]: I'm fine with it as long as external people can't access these docs.
We need to cater to our internal teams as best as possible.

Comment (czubair) [reply .github/PULL_REQUEST_TEMPLATE.md:61]: @weshaggard it seems these links would work as we need them to - to reach internal folks only.

Comment (weshaggard) [reply .github/PULL_REQUEST_TEMPLATE.md:61]: @JeffreyRichter is there any reason these cannot or shouldn't be in our public specs repo?

Comment (JeffreyRichter) [reply .github/PULL_REQUEST_TEMPLATE.md:61]: Yes, the http://aka.ms/bcforapi PDF file has employee names in it related to legal and marketing and who signs off on breaking changes.

The wiki has internal comments/discussion about proposed changes to policies which must not be external.

Comment (weshaggard) [reply .github/PULL_REQUEST_TEMPLATE.md:61]: OK so there might be some things we need to protect but for an open source project there are a lot of links that are going to 404 in this common PR template which is not a great experience.  As a potential future improvement, it would be good to separate these into public and private and keep the private links to a minimum (ideally no more than 1).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
...
+  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>
+  
+### Tooling
+ * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
+ * [Spectral Linting](https://aka.ms/style)
+ * [Open API Hub](https://aka.ms/openapiportal)
+
+### Guidelines & Specifications
+ * [Azure REST API Guidelines](https://aka.ms/guidelines)
```
Comment (weshaggard) [inline .github/PULL_REQUEST_TEMPLATE.md:118]: @czubair have you validated all these links? I haven't check them all but at least this one doesn't work.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
...
+* :love_letter: [email](mailto://azureapirbcore@microsoft.com)
+
+</details>
+
+<details>
+  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>
+  
+### Tooling
+ * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
+ * [Spectral Linting](https://aka.ms/style)
```
Comment (weshaggard) [inline .github/PULL_REQUEST_TEMPLATE.md:114]: I don't think this is the correct link, also why are we linking spectral? Aren't we still using OAV in our pipelines?

Comment (mikekistler) [reply .github/PULL_REQUEST_TEMPLATE.md:114]: OAV is used in our pipelines but we use Spectral in the API reviews (in addition to all the checks done in the pipeline) so we want teams to be aware of it.

Comment (mikekistler) [reply .github/PULL_REQUEST_TEMPLATE.md:114]:
```suggestion
 * [Spectral Linting](https://aka.ms/azapi/style)
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (mikekistler) [inline .github/PULL_REQUEST_TEMPLATE.md:118]:
```suggestion
 * [Azure REST API Guidelines](https://aka.ms/azapi/guidelines)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
...
+ * [Open API Hub](https://aka.ms/openapiportal)
+
+### Guidelines & Specifications
+ * [Azure REST API Guidelines](https://aka.ms/guidelines)
+ * [OpenAPI Style Guidelines](https://aka.ms/style)
```
Comment (mikekistler) [inline .github/PULL_REQUEST_TEMPLATE.md:119]:
```suggestion
 * [OpenAPI Style Guidelines](https://aka.ms/azapi/style)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
...
+This PR template is for Data Plane APIs only.
+
+## API Info: The Basics
+Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).
+
+* Link to engagement record issue: 
+
+Is this review for (select one):
+
+- [ ] a private preview
```
Comment (mikekistler) [inline .github/PULL_REQUEST_TEMPLATE.md:83]: Should we even list "private preview" as an option for the public repo?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
...
+
+#### Description
+< Your text here >
+  
+#### References & Links  
+* Design Document: 
+* Previous Open API Doc: 
+* Updated paths:
+  
+## :grey_question: Got questions? Need additional info?? We are here to help!
```
Comment (mikekistler) [inline .github/PULL_REQUEST_TEMPLATE.md:98]: Should be subheading of "Data Plane API"
```suggestion
### :grey_question: Got questions? Need additional info?? We are here to help!
```

**Review by mikekistler — APPROVED** [review]
This looks great. 👍 

I left a few suggested changes and one question, but all minor points.

**Comment by akning-ms** [comment]
+ @josefree to review before merge

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
...
+This PR template is for Data Plane APIs only.
+
+### API Info: The Basics
+Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).
+
+* Link to engagement record issue: 
+
+Is this review for (select one):
+
+- [ ] a private preview
```
Comment (czubair) [inline .github/PULL_REQUEST_TEMPLATE.md:83]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -22,7 +28,7 @@ Add a changelog entry for this PR by answering the following questions:
       - [ ] Terraform
       - [ ] No refresh required for updates in this PR
 
-### Contribution checklist:
+### Contribution checklist (MS Employees Only):
 - [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
```
Comment (mikekistler) [inline .github/PULL_REQUEST_TEMPLATE.md:32]: The "bcforapi" short link points to an outdated version of the breaking changes doc. I suggest using the link that Jeffrey/Srikanth maintain to always point to the latest version.
```suggestion
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -39,24 +45,82 @@ If any further question about AME onboarding or validation tools, please view th
...
 ### Breaking Change Review Checklist 
-If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
-- [ ] Removing API(s) in a stable version
-- [ ] Removing properties in a stable version
-- [ ] Removing API version(s) in a stable version
-- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
-- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
-
-**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+If you have any breaking changes as defined in the [Breaking Change Policy](http://aka.ms/bcforapi), request approval from the Breaking Change Review Board.
```
Comment (mikekistler) [inline .github/PULL_REQUEST_TEMPLATE.md:59]: Here too.
```suggestion
If you have any breaking changes as defined in the [Breaking Change Policy](http://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 11]
```diff
@@ -39,24 +45,82 @@ If any further question about AME onboarding or validation tools, please view th
...
+  
+### Tooling
+ * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
+ * [Spectral Linting](https://aka.ms/azapi/style)
+ * [Open API Hub](https://aka.ms/openapiportal)
+
+### Guidelines & Specifications
+ * [Azure REST API Guidelines](https://aka.ms/azapi/guidelines)
+ * [OpenAPI Style Guidelines](https://aka.ms/azapi/style)
+ * [Azure Breaking Change Policy](http://aka.ms/bcforapi)
```
Comment (mikekistler) [inline .github/PULL_REQUEST_TEMPLATE.md:119]: Here too.
```suggestion
 * [Azure Breaking Change Policy](http://aka.ms/AzBreakingChangesPolicy/)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 12]
```diff
@@ -39,24 +45,82 @@ If any further question about AME onboarding or validation tools, please view th
...
+
+## Data Plane API
+
+<details>
+  <summary>Expand this section for dataplane</summary>
+
+This PR template is for Data Plane APIs only.
+
+### API Info: The Basics
+Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).
```
Comment (czubair) [inline .github/PULL_REQUEST_TEMPLATE.md:77]:
```suggestion

```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 13]
```diff
@@ -39,24 +45,81 @@ If any further question about AME onboarding or validation tools, please view th
...
+## Data Plane API
+
+<details>
+  <summary>Expand this section for dataplane</summary>
+
+This PR template is for Data Plane APIs only.
+
+### API Info: The Basics
+
+* Link to engagement record issue:
```
Comment (czubair) [inline .github/PULL_REQUEST_TEMPLATE.md:78]:
```suggestion
* Link to engagement record issue (if known): 
```

**Review by lmazuel — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 14]
```diff
@@ -39,24 +45,81 @@ If any further question about AME onboarding or validation tools, please view th
...
+  
+**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+  
+NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
 
 Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).
+  
+</details>
+
+## non-ARM/ Data Plane API
```
Comment (czubair) [inline .github/PULL_REQUEST_TEMPLATE.md:69]:
```suggestion
## Non-ARM API Information (Data Plane)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 15]
```diff
@@ -39,24 +43,75 @@ If any further question about AME onboarding or validation tools, please view th
 > - All removals
 
 Otherwise your PR may be subject to ARM review requirements. Complete the following:
-- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
+- [ ] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
```
Comment (raych1) [inline .github/PULL_REQUEST_TEMPLATE.md:46]: Hi @czubair , can you revert the change of quotation mark coz it will break the automation tool?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 16]
```diff
@@ -39,24 +43,75 @@ If any further question about AME onboarding or validation tools, please view th
...
+
+This PR template is for Data Plane APIs only.
+
+### API Info: The Basics
+
+* Link to engagement record issue (if known): 
+
+Is this review for (select one):
+
+- [ ] a public preview
```
Comment (weidongxu-microsoft) [inline .github/PULL_REQUEST_TEMPLATE.md:76]:
```suggestion
- [ ] A public preview
```

Comment (weidongxu-microsoft) [reply .github/PULL_REQUEST_TEMPLATE.md:76]: Could there be a private preview? I assume this template will be shared in private repo as well, and team may use that repo for private preview.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 17]
```diff
@@ -39,24 +43,75 @@ If any further question about AME onboarding or validation tools, please view th
...
-
-**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+If you have any breaking changes as defined in the [Breaking Change Policy](http://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.
+  
+**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+  
+NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
```
Comment (weidongxu-microsoft) [inline .github/PULL_REQUEST_TEMPLATE.md:61]: Is this message important (as the link seems also private)?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 18]
```diff
@@ -39,24 +43,75 @@ If any further question about AME onboarding or validation tools, please view th
...
+
+### API Info: The Basics
+
+* Link to engagement record issue (if known): 
+
+Is this review for (select one):
+
+- [ ] a public preview
+- [ ] GA release 
+
```
Comment (weidongxu-microsoft) [inline .github/PULL_REQUEST_TEMPLATE.md:78]: Should we also add a check list like below?

```
### REST API Review Checklist 

- [ ] Ensure you've reviewed [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). This is required before you can request review from Azure REST API Stewardship.
```

I see it is mentioned once in `ARM API Information` section and also included indirectivity in contact section. But we might want to emphasis the Guidelines again.

**Review by weidongxu-microsoft — APPROVED** [review]
LGTM.

Please update according to Ray's comment.

PS: dev rarely ever fill anything in the template. Many of them not even check the checkbox correctly. :-(

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 19]
```diff
@@ -39,24 +43,75 @@ If any further question about AME onboarding or validation tools, please view th
 > - All removals
 
 Otherwise your PR may be subject to ARM review requirements. Complete the following:
-- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
+- [ ] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
   - Adding a new service
   - Adding new API(s)
   - Adding a new API version
-    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).
+    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.
```
Comment (akning-ms) [inline .github/PULL_REQUEST_TEMPLATE.md:50]: Besides adding this detail guide, Can we add back Openapi hub link here which can help service team easy to create this kind of PR?

**Comment by weidongxu-microsoft** [comment]
@josefree 
Hi Josephine, would you share your opinion? Thanks.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 15 above)
Comment (czubair) [inline .github/PULL_REQUEST_TEMPLATE.md:46]:
```suggestion
- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 19 above)
Comment (czubair) [inline .github/PULL_REQUEST_TEMPLATE.md:50]:
```suggestion
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 20]
```diff
@@ -43,20 +47,71 @@ Otherwise your PR may be subject to ARM review requirements. Complete the follow
...
 
 - [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.
```
Comment (czubair) [inline .github/PULL_REQUEST_TEMPLATE.md:52]:
```suggestion
- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc). This is required before you can request review from ARM API Review board.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 17 above)
Comment (mikekistler) [reply .github/PULL_REQUEST_TEMPLATE.md:54]: I think this policy is not widely / well understood so we should keep it.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (josefree) [reply .github/PULL_REQUEST_TEMPLATE.md:52]: Hi Wes, I agree with you.  We can prioritize the efforts when we working on to support 3rd party resource provider.

**Review by josefree — APPROVED** [review]

**Review by raych1 — APPROVED** [review]

**Comment by JackTn** [comment]
Hi, @czubair. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Comment by JackTn** [comment]
Hi, @czubair. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Comment by JackTn** [comment]
Hi, @czubair. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.
